packages/block-library/src/spacer/edit.js

```diff
@@ -48,45 +52,27 @@ const SpacerEdit = ( {
 					bottomLeft: false,
 					topLeft: false,
 				} }
+				isSelected={ isSelected }
 				onResizeStart={ onResizeStart }
 				onResizeStop={ ( event, direction, elt, delta ) => {
 					onResizeStop();
 					const spacerHeight = parseInt( height + delta.height, 10 );
 					setAttributes( {
 						height: spacerHeight,
 					} );
-					setInputHeightValue( spacerHeight );
 				} }
 			/>
 			<InspectorControls>
 				<PanelBody title={ __( 'Spacer settings' ) }>
-					<BaseControl label={ __( 'Height in pixels' ) } id={ id }>
-						<input
-							type="number"
-							id={ id }
-							onChange={ ( event ) => {
-								let spacerHeight = parseInt(
-									event.target.value,
-									10
-								);
-								setInputHeightValue( spacerHeight );
-								if ( isNaN( spacerHeight ) ) {
-									// Set spacer height to default size and input box to empty string
-									setInputHeightValue( '' );
-									spacerHeight = 100;
-								} else if ( spacerHeight < 20 ) {
-									// Set spacer height to minimum size
-									spacerHeight = 20;
-								}
-								setAttributes( {
-									height: spacerHeight,
-								} );
-							} }
-							value={ inputHeightValue }
-							min="20"
-							step="10"
-						/>
-					</BaseControl>
+					<RangeControl
```

Comment:
Seems like we're changing this component for the web as well, would be nice to have the approval from a core dev here

Reply:
Not a core dev, but this looks good to me. In my opinion, sliders should be used for controls where there is a large range of possible values and the exact value doesn't really matter that much. This seems like a perfect place to use a slider.